lib/Runtime/Library/DateImplementation.cpp

```diff
@@ -404,7 +404,7 @@ namespace Js {
         // Add the year.
         if (pymd->year > 0)
         {
-            bs->AppendChars(pymd->year, 10, ConvertLongToString);
+            bs->AppendChars(static_cast<uint32>(pymd->year), 10, ConvertUInt32ToString_ZeroPad_4);
```

Comment:
Did something change that now this static_cast is necessary?

Reply:
I remember I put `static_cast` there only for stylistic reason (making `int` to `uint32` conversion explicit). It's unnecessary so I removed it.